src/api/parser/src/routes/parser.js

```diff
@@ -0,0 +1,9 @@
+const { Router, logger } = require('@senecacdot/satellite');
+
+const router = Router();
+
+router.get('/', (req, res) => {
+  res.status(200).send('hi');
```

Comment:
This seems not worth doing.  Should we move what you have at `/queues` to this instead?  That is, going to `http://localhost/v1/parser` would show me the Bull Queues.

This also implies that we need some authentication/authorization (see Satellite, which has middleware you can use for this).  We either need to require admin access, or maybe just authenticated users can see it?  I'm not sure how sensitive this is.

Reply:
Sure, we can disregard the whole `parser.js` file and just do `service.router.use('/', bullRouter)` in `index.js`